api/timeline/v1/timeseries.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package clutch.timeline.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/timeline/v1;timelinev1";
+
+import "google/protobuf/any.proto";
+import "validate/validate.proto";
+
+// A timeseries datapoint message is useful for organizing events
+// to be displayed in a timeline view.  Users can transform data
+// into a timeseries format and be able to organize them.
+message TimeseriesData {
+  string project_name = 1 [ (validate.rules).string = {min_bytes : 1} ];
+  // For a range, both start time and end time will be present
+  // For a singular event, only start time will be filled in.
+  int64 start_time_millis = 2 [ (validate.rules).int64.gt = 0 ];
+  int64 end_time_millis = 3;
+  // The plan here is to have each user of this proto to put their unique data in a proto
```

Comment:
nit: probably would change this to either be more concrete or to be written as a TODO